roxie/ccd/ccdserver.cpp

```diff
@@ -418,7 +418,7 @@ class CRoxieServerActivityFactoryBase : public CActivityFactory, implements IRox
     bool optUnstableInput = false;  // is the input forced to unordered?
     bool optUnordered = false; // is the output specified as unordered?
     unsigned heapFlags;
-
+    bool isCodeSigned = false;
```

Comment:
minor: moving this before heapFlags would reduce memory consumption by 8 bytes (since it would pack in with the other booleans)